src/solvers/refinement/string_refinement.cpp

```diff
@@ -231,28 +229,6 @@ static void display_index_set(
          << " newly added)" << eom;
 }
 
-/// compute the index set for all formulas, instantiate the formulas with the
-/// found indexes, and add them as lemmas.
-
-static void display_current_index_set(
```

Comment:
I think @romainbrenguier should comment on whether this is in active use while debugging string refinement behaviour.

Reply:
They were used right next to each other, and the `display_index_set` function highlights which indices are new.